x-pack/plugins/integrations_manager/index.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { resolve } from 'path';
+import {
+  CoreSetup,
+  LegacyPluginInitializer,
+  PluginInitializerContext,
+  Server,
+} from './common/types';
+import { Plugin } from './server/plugin';
+import { ID } from './common/constants';
+import { mappings, savedObjectSchemas } from './server/saved_objects';
+
+const ICON = 'merge';
+const PREFIX = `xpack.${ID}`;
+const ROOT = `plugins/${ID}`;
+const pluginTitle = i18n.translate(`${PREFIX}.pluginTitle`, {
```

Comment:
@jfsiii  Regarding string localization, please follow localization guidelines: [kbn-i18n/GUIDELINE](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md), [kbn-i18n/README](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/README.md),  [i18n/README](https://github.com/elastic/kibana/blob/master/src/dev/i18n/README.md).
Please pay your attention that translations IDs can be a string literal only, each plugin has a prefix for ID that should be added to [.i18nrc.json](https://github.com/elastic/kibana/blob/master/.i18nrc.json) (e.g. for this plugin it can be `xpack.integrationManager`).

cc/ @Bamieh

Reply:
@maryia-lapata Thanks for the links and good eyes. I made the changes you suggested in https://github.com/elastic/kibana/pull/37950/commits/05561695413d7e3565eb2a8d9a129732e0bcbaba